A/AMGCL_C/build_tarballs.jl

```diff
@@ -0,0 +1,82 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+
+#
+# This script builds [AMGCL_C](https://github.com/j-fu/amgcl_c)
+# which is a C wrapper to [AMGCL](https://github.com/ddemidov/amgcl),
+# an algebraic multigrid (AMG) based iterative solver library.
+#
+# Both AMGCL and AMGCL_C are MIT licensed.
+#
+# During the build, AMGCL_C downloads AMGCL via ExternalProject_Add() 
+#
+# CMake parameters:
+#   BUILD_DI_INTERFACE: Build interface for double + int
+#   BUILD_DL_INTERFACE: Build interface for double + long
+#   BLOCKSIZES: List of blocksizes instantiated for static blocking in AMGCL
+#               More blocksizes => longer compile time, larger lib.
+#               (see e.g. https://amgcl.readthedocs.io/en/latest/tutorial/Serena.html)
+#
+# Handling of integer types attempts to be similar to umfpack: build both DL and DI on
+# 64 bit systems and only DI on 32bit. See
+# https://github.com/JuliaSparse/SparseArrays.jl/blob/feb54ee5e49008bd157227099cafe604a67c36fb/src/solvers/umfpack.jl#L145
+# https://github.com/JuliaSparse/SparseArrays.jl/blob/feb54ee5e49008bd157227099cafe604a67c36fb/src/solvers/umfpack.jl#L578
+#
+using BinaryBuilder, Pkg
+
+name = "AMGCL_C"
+version = v"0.1.0"
```

Comment:
Your comment says 0.1.2 below, but this version is 0.1.0, and there doesn't appear to be a 0.1.2 release yet (the below commit SHA is just the most recent commit on main.

It would be good to do another release and then update this version number before we merge this.

Reply:
Relased AMGCL_C 0.1.2 and set script version to 0.1.2 .